resources/web/wwi/webots.js

```diff
@@ -190,7 +190,8 @@ webots.View = class View {
           this.server = new Server(this.url, this, callback);
           this.server.connect();
         } else { // url expected form: "ws://cyberbotics2.cyberbotics.com:80"
-          this.stream = new Stream(this.url, this, finalizeWorld);
+          var httpServerUrl = this.url.replace(/ws/, 'http'); // Serve the texture images.
```

Comment:
This seems to work fine with http, but not https. We should quickly extend it to https as well (possibly in a separate PR) otherwise, it will be a regression.

Reply:
This is deserving a comment, but this line supports `https` (it replaces the first `ws` by `http`, so it can change `wss` => `https`). Nevertheless, I need to test more seriously SSL connections.